### DISCUSSION TITLE
gapps question

### DISCUSSION THREAD

**Discussion by kmarzic** [bakkeby/dwm-flexipatch#212] [Q&A] [answered]
Hi,

Do you know why after some time gapps are not equals, and they start to "grow" on some tag:

![screenshot_20220111 132017](https://user-images.githubusercontent.com/4183537/148941677-ecd74337-e303-4596-83ab-1235119c9c5b.png)

![screenshot_20220111 132034](https://user-images.githubusercontent.com/4183537/148941696-3c216f3d-eac7-4156-94d7-3849e3c5bd5c.png)

I am using the following patches:

```
$ cat patches.h | grep "^#define" | grep "1$"
#define BAR_AWESOMEBAR_PATCH 1
#define BAR_LTSYMBOL_PATCH 1
#define BAR_STATUS_PATCH 1
#define BAR_STATUS2D_PATCH 1
#define BAR_SYSTRAY_PATCH 1
#define BAR_TAGS_PATCH 1
#define BAR_WINICON_PATCH 1
#define BAR_WINTITLE_PATCH 1
#define BAR_TITLE_LEFT_PAD_PATCH 1
#define BAR_ALPHA_PATCH 1
#define BAR_BORDER_PATCH 1
#define BAR_IGNORE_XFT_ERRORS_WHEN_DRAWING_TEXT_PATCH 1
#define BAR_PADDING_PATCH 1
#define BAR_STATUSPADDING_PATCH 1
#define ATTACHASIDE_PATCH 1
#define CENTER_PATCH 1
#define CYCLELAYOUTS_PATCH 1
#define MONITOR_RULES_PATCH 1
#define MOVESTACK_PATCH 1
#define PERTAG_PATCH 1
#define PERTAG_VANITYGAPS_PATCH 1
#define SAVEFLOATS_PATCH 1
#define SCRATCHPADS_PATCH 1
#define VANITYGAPS_PATCH 1
#define VANITYGAPS_MONOCLE_PATCH 1
#define DECK_LAYOUT 1
#define HORIZGRID_LAYOUT 1
#define TILE_LAYOUT 1
#define MONOCLE_LAYOUT 1
```

Best regards

**Comment by bakkeby** [comment]
Gaps between what exactly?

If it is specifically about gaps between windows and for the window area border (vanitygaps) then you have `PERTAG_VANITYGAPS_PATCH` enabled which means that the gaps are stored on a per tag basis.

Possibly you are doing some keybindings that also increases the gaps, which is easy to do if you have changed the MODKEY to be `Mod4Mask` (Super) as the default keybindings may then conflict.

**Comment by kmarzic** [comment]
Hi,
Thanks for your reply.
My problem is the gap between screen border and window border.
Yes, I switched MODKEY from Mod1Mask (Alt) to Mod4Mask (Super) - maybe that is the reason the gaps are increasing.

**Comment by bakkeby** [comment, marked as answer]
Maybe have a look at these:

https://github.com/bakkeby/dwm-flexipatch/blob/0404e12dbaca203325f391f751f1188f91a7ef1a/config.def.h#L942-L957

If you use MOD+i or MOD+o then the gaps would increase yes. You can change all the `Mod4Mask` references to `Mod1Mask` to have the original behaviour after changing the MODKEY.

**Comment by kmarzic** [comment]
Hi,
Thanks a lot!
Yes, this what was causing problems in my configuration
